src/easydmp/dmpt/signals.py

```diff
@@ -0,0 +1,8 @@
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+
+
+@receiver(pre_save, sender='dmpt.Question')
+def prohibit_0_question_position(sender, instance, **kwargs):
```

Comment:
@hmpf Do we have a way to do this nicer? I haven't found something like QuestionForm clean() around here, is there something like that?

Reply:
We could swap out the autogenerated form in the admin maybe. Though whenever there is a writeable API for making questions, the check would have to move earlier.

Reply:
I fixed this.